File: MAR18/BIGSALE.py
def main():
	for test in range(int(input().strip())):
		loss=0
		a=[]
		n=int(input().strip())
		for i in range(n):
			a.append([int(x) for x in input().strip().split()])
		for i in range(n):
			temp=(a[i][0]*a[i][1])+(a[i][0]*a[i][1])*(a[i][2]/100.0)
			temp-=temp*(a[i][2]/100.0)
			loss+=(a[i][0]*a[i][1])-temp
		print(loss)
if __name__ == '__main__':
	main()

File: MAR18/BIGSALE.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
	int t;
	cin>>t;
	while(t--)
	{
		int n;
		float temp,loss=0;
		cin>>n;
		int **a=new int *[n];
		for (int i = 0; i < n; ++i)
			a[i] = new int [3];
		for(int i=0;i<n;i++)
			cin>>a[i][0]>>a[i][1]>>a[i][2];
		for(int i=0;i<n;i++)
		{
			temp=(a[i][0]*a[i][1])+(a[i][0]*a[i][1])*(a[i][2]/100.0);
			temp-=temp*(a[i][2]/100.0);
			loss+=(a[i][0]*a[i][1])-temp;
		}
		printf("%0.8f",loss);	
	}

return 0;
}